Prace_domowe/Praca_domowa4/Grupa4/Marcin Wilk/PD4.ipynb

```diff
@@ -0,0 +1,1294 @@
+{
```

Comment:
<p>Wyraźnie widzimy niestabilność modelu wyskalowanego. 60 pp. rozrzutu, to jest już coś. Jakbyś to przeanalizował, to by było suuuper</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>